opentelemetry-otlp/tests/integration_test/expected/serialized_traces.json

```diff
@@ -0,0 +1,133 @@
+        "droppedAttributesCount": 0
+      },
+      "scopeSpans": [
+        {
+          "scope": {
+            "name": "ex.com/basic",
+            "version": "",
+            "attributes": [],
+            "droppedAttributesCount": 0
+          },
+          "spans": [
+            {
+              "traceId": "9b458af7378cba65253d7042d34fc72e",
+              "spanId": "cd7cf7bf939930b7",
+              "traceState": "",
+              "parentSpanId": "d58cf2d702a061e0",
+              "flags": 0,
+              "name": "Sub operation...",
+              "kind": 1,
+              "startTimeUnixNano": "1703985537070566698",
+              "endTimeUnixNano": "1703985537070572718",
+              "attributes": [
+                {
+                  "key": "lemons",
+                  "value": {
+                    "stringValue": "five"
+                  }
+                }
+              ],
+              "droppedAttributesCount": 0,
+              "events": [
+                {
+                  "timeUnixNano": "1703985537070567697",
+                  "name": "Sub span event",
+                  "attributes": [],
+                  "droppedAttributesCount": 0
+                }
+              ],
+              "droppedEventsCount": 0,
+              "links": [],
+              "droppedLinksCount": 0,
+              "status": {
+                "message": "",
+                "code": 0
+              }
+            }
+          ],
+          "schemaUrl": ""
+        }
+      ],
+      "schemaUrl": ""
+    },
+    {
+      "resource": {
+        "attributes": [
+          {
+            "key": "service.name",
+            "value": {
+              "stringValue": "basic-otlp-tracing-example"
+            }
+          }
+        ],
+        "droppedAttributesCount": 0
+      },
+      "scopeSpans": [
+        {
+          "scope": {
+            "name": "ex.com/basic",
+            "version": "",
+            "attributes": [],
+            "droppedAttributesCount": 0
+          },
+          "spans": [
+            {
+              "traceId": "9b458af7378cba65253d7042d34fc72e",
+              "spanId": "d58cf2d702a061e0",
+              "traceState": "",
+              "parentSpanId": "",
+              "flags": 0,
+              "name": "operation",
+              "kind": 1,
+              "startTimeUnixNano": "1703985537070558635",
+              "endTimeUnixNano": "1703985537070580454",
+              "attributes": [
+                {
+                  "key": "ex.com/another",
+                  "value": {
+                    "stringValue": "yes"
+                  }
+                }
+              ],
+              "droppedAttributesCount": 0,
+              "events": [
+                {
+                  "timeUnixNano": "1703985537070563326",
+                  "name": "Nice operation!",
+                  "attributes": [
+                    {
+                      "key": "bogons",
+                      "value": "100"
```

Comment:
The string value here should use `Value::String` during serialization instead of plain string for attributes.  When a number is preferred on deserialization, then it should serialized as such.